api/config/config_local.json

```diff
@@ -23,7 +23,10 @@
     "bufferCapacity": 2,
     "bufferRefillProjectsPerTask": 1,
     "defaultFreeCreditsDollarLimit": 300.0,
-    "freeTierCostAlertThresholds": [0.5, 0.75],
+    "freeTierCostAlertThresholds": [
```

Comment:
Welp.  The good thing about autoformatters is that they only run once.

Reply:
Which autoformatter did you run? I'm wondering if we can document something somewhere about how this should be done, so we can have consistent adoption.

Reply:
IntelliJ Save Action, following our onboarding/setup doc: https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit#heading=h.sjqyvxq4luy4

There are many options for Save Actions however, and I was initially quite overzealous on a different branch and had to revert some changes.